random-number-challenge/script.js

```diff
@@ -0,0 +1,21 @@
+// Collect input from a user
+const inputLow = prompt('Please provde your lowest number')
+const inputHigh = prompt ('Please provide your highest number') ;
+
+// Convert the input to a number
+const lowNumber = parseInt(inputLow);
+const highNumber = parseInt (inputHigh) ;
+
+
+
+if (highNumber) {
```

Comment:
This will fail if the `lowNumber` is empty. Add safeguard to `lowNumber` as well, add it on this condition.

Reply:
Hi @xianvalencia , added `lowNumber` on the condition as well